Bottom: 0313cc95380e8433e33705c1801e95f98f485d2f
Top:    511d59a4118b97b791c3953b1998887da29858e2
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-16 15:12:07 +0800

Refresh of autofs-5.0.7-only-prove-specific-nfs-version-if-requested.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index 64546d7..a1e2d40 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -54,18 +54,6 @@ int mount_init(void **context)
 	return !mount_bind;
 }
 
-unsigned int set_nfs_vers(const char *fstype)
-{
-	unsigned int mount_default_proto = defaults_get_mount_nfs_default_proto();
-	unsigned int vers = NFS_VERS_MASK | NFS_PROTO_MASK;
-	if (strcmp(fstype, "nfs4") == 0)
-		vers = NFS4_VERS_MASK | TCP_SUPPORTED;
-	else if (mount_default_proto == 4)
-		vers = vers | NFS4_VERS_MASK;
-
-	return vers;
-}
-
 int mount_mount(struct autofs_point *ap, const char *root, const char *name, int name_len,
 		const char *what, const char *fstype, const char *options,
 		void *context)
